src/components/ModelizerModelView.vue

```diff
@@ -141,3 +141,9 @@ onUnmounted(() => {
     width: 100%;
   }
 </style>
+
+<style lang="scss">
+div#root svg {
```

Comment:
Add comment to explain why this had to be done, ex  `Override quasar style for svg`